_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp

```diff
@@ -1650,6 +1678,16 @@ mfxStatus CheckVideoParam(MfxVideoParam& par, MFX_ENCODE_CAPS_HEVC const & caps,
 
     switch (CO3.TargetBitDepthLuma)
     {
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+    case 12:
+        invalid += CheckOption(CO3.TargetBitDepthChroma, 12/*, 10, 8*/, 0);
+        invalid += CheckOption(par.mfx.CodecProfile
+//            , (mfxU16)MFX_PROFILE_HEVC_MAIN10
```

Comment:
what this commented line is about?

Reply:
For 12b encoding both Luma and Chroma TargetBitdepth should be 12 (at least other behavior is not confirmed yet) while for 10b encoding it is allowed to have LumaTargetBitdepth=10 and ChromaTargetBitdepth=8
Commented lines can be easily removed.